libraries/networking/src/udt/SendQueue.h

```diff
@@ -42,21 +42,8 @@ class SendQueue : public QObject {
     Q_OBJECT
     
 public:
-    
-    class DoubleLock {
-    public:
-        DoubleLock(std::mutex& mutex1, std::mutex& mutex2) : _mutex1(mutex1), _mutex2(mutex2) { lock(); }
-        ~DoubleLock() { unlock(); }
-        
-        DoubleLock(const DoubleLock&) = delete;
-        DoubleLock& operator=(const DoubleLock&) = delete;
-        
-        void lock() { std::lock(_mutex1, _mutex2); }
-        void unlock() { _mutex1.unlock(); _mutex2.unlock(); }
-    private:
-        std::mutex& _mutex1;
-        std::mutex& _mutex2;
-    };
+    using clock = std::chrono::high_resolution_clock;
```

Comment:
I actually did that so you don't have to worry about what clock to use, it's specified in the class definition.
Turns out `high_resolution_clock` is already an alias to either `system_clock` or `steady_clock`.
I can replace `clock` by `high_resolution_clock`, I think this is a better approach than to do a `using namespace std::chrono` at the top of the cpp.